README.md

```diff
@@ -109,34 +106,6 @@ In order to test caching locally,
 
 To run stress testing against the Salesforce instance, refer to the documentation in the [stress testing folder](load_testing/load_testing.md)
 
-## Translations process with [Phrase](https://app.phrase.com/accounts/city-county-of-san-francisco/projects/dahlia-sf-dahlia-web/dashboard)
```

Comment:
Moved to https://sfgovdt.jira.com/wiki/spaces/HOUS/pages/3189538929/Updating+translations+through+Phrase